ui/narrative/methods/import_genbank_as_genome_from_staging/spec.json

```diff
@@ -194,10 +202,6 @@
           "input_parameter": "scientific_name",
           "target_property": "scientific_name"
         },
-        {
-          "input_parameter": "taxon_id",
```

Comment:
Can you explain why `taxon_id` is removed? It seems like you want to keep it and just assign `ncbi_taxon_id` to it. Is `ncbi_taxon_id` and `taxon_id` the same thing? Did GFU get changed to take `ncbi_taxon_id` instead of `taxon_id`?

Reply:
GFU is not changed to take ncbi_taxon_id instead of taxon_id. So yes, I am keep taxon_id to assign ncbi_taxon_id to taxon_id.  This removal of taxon_id is only for the ui portion, because the dynamic dropdown will fetch the ncbi_taxon_id that could conflict with the taxon_id input.